cmake/modules/BuildRocksDB.cmake

```diff
@@ -60,15 +61,23 @@ function(build_rocksdb)
   set(rocksdb_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/rocksdb")
   set(rocksdb_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/rocksdb")
   set(rocksdb_LIBRARY "${rocksdb_BINARY_DIR}/librocksdb.a")
+  if(CMAKE_MAKE_PROGRAM MATCHES "make")
```

Comment:
Don't see why you need this distinction?
Gmake also understands parallel options?

Reply:
I think it does, not quite sure we need these distinctions, can update them as well to just use $(MAKE) straightforward. @tchaikov should we update it in BuildSPDK.cmake BuildDPDK.cmake as well?